src/exit/exit.rs

```diff
@@ -0,0 +1,28 @@
+use c_types::c_int;
+use super::_Exit::_Exit;
+
+#[linkage = "weak"]
+#[no_mangle]
+pub unsafe extern "C" fn __funcs_on_exit() {}
+
+#[linkage = "weak"]
+#[no_mangle]
+pub unsafe extern "C" fn __stdio_exit() {}
+
+#[linkage = "weak"]
+#[no_mangle]
+pub unsafe extern "C" fn _fini() {}
+
+#[linkage = "weak"]
+#[no_mangle]
+pub unsafe extern "C" fn __libc_exit_fini() {
+    _fini()
```

Comment:
I think I'd rather add a TODO: those symbols are used by dynamic linker for purposes that are still mysterious to me. You can read about Rich Felker's motivations in his very descriptive commit messages here: http://git.musl-libc.org/cgit/musl/log/src/exit/exit.c